lofo/lofo_importance.py

```diff
@@ -9,72 +9,66 @@
 
 class LOFOImportance:
 
-    def __init__(self, df, features, target,
-                 scoring, model=None, cv=4, n_jobs=None):
+    def __init__(self, dataset, scoring, model=None, fit_params=None, cv=4, n_jobs=None):
```

Comment:
I'm not sure users will be happy that it's not backwards compatible.. (dataset instead of df)

Reply:
Unfortunately, it is not backward compatible indeed. Now, they first create datasets which have pandas df, features, target etc.
https://github.com/aerdem4/lofo-importance/blob/feature-groups/lofo/dataset.py

Reply:
Yes.. do you want to add a line in the README to explicitly mention that it's not backwards compatible? Or maybe a temporary error printout if they use a pandas df in the function call? Up to you.

Reply:
I have updated README and Example Notebook to show how it works with the new dataset objects. The change may be annoying for the users indeed, but at this point, the project is in beta state (we don't have regular users yet probably).